Resources/Prototypes/Entities/Mobs/base.yml

```diff
@@ -238,3 +238,32 @@
     bloodlossHealDamage:
       types:
         Bloodloss: -1
+
+# Used for robotic mobs, like hivebots and cyborgs
+- type: entity
+  save: false
+  id: MobRobotic
```

Comment:
this really needs to be combined or built into `MobSiliconBase`. The other one looks a bit too specialized but this is really weird parenting